src/main/java/org/amahi/anywhere/activity/AuthenticationActivity.java

```diff
@@ -59,13 +63,47 @@
 {
 	@Inject
 	AmahiClient amahiClient;
-
+  ImageView imageView;
+  
 	@Override
 	protected void onCreate(Bundle savedInstanceState) {
 		super.onCreate(savedInstanceState);
 		setContentView(R.layout.activity_authentication);
 
-		setUpInjections();
+    getUsernameEdit().setOnTouchListener(new View.OnTouchListener() {
+
+            @Override
+            public boolean onTouch(View view, MotionEvent motionEvent) {
+                getUsernameEdit().setOnFocusChangeListener(new View.OnFocusChangeListener() {
```

Comment:
Setting a listener inside another listener? seriously?
This will make a new copy of focus listener every time you touch the EditText. This can be simplified! Please revise your code.